exporters/otlp/otlpmetric/otlpmetricgrpc/client.go

```diff
@@ -105,3 +326,4 @@ func (c *client) UploadMetrics(ctx context.Context, protoMetrics []*metricpb.Res
 	}
 	return err
 }
```

Comment:
Should we remove these commented codes?

Reply:
Gah, yes indeed. Thanks for catching this.